### ISSUE TITLE
AlwaysAcknowledgeGatewayInteractions = true breaks Message components

### ISSUE THREAD

**Issue by GedasFX** [Discord-Net-Labs/Discord.Net-Labs#97]
## Context
Version: Discord.Net.Labs 2.4.9

## Problem
It appears that setting `AlwaysAcknowledgeInteractions = true` makes it ignore message components. Changing this to false made them appear again.

It could be an issue with acknowledgement, but due to #95 was unable to test it.

Used command:
```
await command.RespondAsync(message, component: new ComponentBuilder()
                                .WithButton("Proceed", "id_1", ButtonStyle.Primary)
                                .WithButton("Cancel", "id_2", ButtonStyle.Danger)
                                .Build());
```

**Comment by Zeiin** [comment]
This was fixed in e8ba24dc0a80bfafeebe245baecd25506fc09a0f (which is in 3.0.0)

**Comment by GedasFX** [comment]
Works in 3.0.1-dev thank you!
